src/uct/ib/base/ib_device.h

```diff
@@ -226,6 +226,7 @@ typedef struct uct_ib_device {
     uint8_t                     pci_cswap_arg_sizes;
     uint8_t                     atomic_align;
     uint8_t                     lag_level;
+    uint8_t                     log_max_ra_req_dc;
```

Comment:
add description `Log (base 2) of the maximum number of outstanding RDMA read/Atomic QP`?
btw, can we remove DC from the name since it is used for RC QPs too?

Reply:
it is not used for RC